python/src/equistore/tensor.py

```diff
@@ -68,6 +78,23 @@ def _from_ptr(ptr):
         obj._blocks = []
         return obj
 
+    @classmethod
+    def _from_pickle(cls, tm_bytes: bytes):
```

Comment:
I had to think a bit on the meaning of `tm_` here, I though it was tmp initiallly

```suggestion
    def _from_pickle(cls, data: bytes):
```